src/content.js

```diff
@@ -254,14 +254,12 @@ function addPatchDiffLinks() {
 		commitUrl = commitUrl.replace(/\/pull\/\d+\/commits/, '/commit');
 	}
 
-	const $commitMeta = $('.commit-meta span.float-right');
-
-	$commitMeta.append(`
+	select('.commit-meta span.float-right').append(
 		<span class="sha-block patch-diff-links">
-			<a href="${commitUrl}.patch" class="sha">patch</a>
-			<a href="${commitUrl}.diff" class="sha">diff</a>
+			<a href={`${commitUrl}.patch`} class="sha">patch</a>
+			<a href={`${commitUrl}.diff`} class="sha">diff</a>
```

Comment:
Ah yeah, that's the *annoyance* I mentioned in https://github.com/sindresorhus/refined-github/pull/558#issuecomment-313317987

Fixing